Tests/StreamChatUITests/SnapshotVariation.swift

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatUI
+import XCTest
+
+enum SnapshotVariation: String, Hashable, CaseIterable {
+    case small
+    case medium
+    case large
+    case extraLarge
+    case extraExtraExtraLarge
+    
+    var traits: UITraitCollection {
+        var traits = [
+            UITraitCollection(displayScale: 1),
+            contentSizeCategoryTrait
+        ]
+        
+        if #available(iOS 12.0, *) {
+            traits.append(userInterfaceStyleTrait)
+        }
+        
+        return UITraitCollection(traitsFrom: traits)
+    }
+    
+    private var contentSizeCategoryTrait: UITraitCollection {
+        switch self {
+        case .small:
+            return UITraitCollection(preferredContentSizeCategory: .small)
+        case .medium:
+            return UITraitCollection(preferredContentSizeCategory: .medium)
+        case .large:
+            return UITraitCollection(preferredContentSizeCategory: .large)
+        case .extraLarge:
+            return UITraitCollection(preferredContentSizeCategory: .extraLarge)
+        case .extraExtraExtraLarge:
+            return UITraitCollection(preferredContentSizeCategory: .extraExtraExtraLarge)
+        }
```

Comment:
I don't think we need that many sizes. Just `small`, `large`, and `extraExtraExtraLarge` is OK imo.